pkgs/applications/networking/browsers/netsurf/buildsystem.nix

```
@@ -1,24 +1,26 @@
-{ lib, stdenv, fetchurl }:
+{ lib
+, stdenv
+, fetchurl
+}:
 
-stdenv.mkDerivation rec {
-  pname = "netsurf-${libname}";
-  libname = "buildsystem";
+stdenv.mkDerivation (finalAttrs: {
+  pname = "netsurf-buildsystem";
   version = "1.9";
 
   src = fetchurl {
-    url = "http://download.netsurf-browser.org/libs/releases/${libname}-${version}.tar.gz";
-    sha256 = "sha256-k4QeMUpoggmiC4dF8GU5PzqQ8Bvmj0Xpa8jS9KKqmio=";
+    url = "http://download.netsurf-browser.org/libs/releases/buildsystem-${finalAttrs.version}.tar.gz";
+    hash = "sha256-k4QeMUpoggmiC4dF8GU5PzqQ8Bvmj0Xpa8jS9KKqmio=";
   };
 
   makeFlags = [
     "PREFIX=$(out)"
   ];
 
-  meta = with lib; {
+  meta = {
     homepage = "https://www.netsurf-browser.org/";
     description = "NetSurf browser shared build system";
-    license = licenses.mit;
-    maintainers = [ maintainers.vrthra maintainers.AndersonTorres ];
-    platforms = platforms.unix;
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ samueldr vrthra AndersonTorres ];
```

```suggestion
    maintainers = with lib.maintainers; [ vrthra AndersonTorres ];
```

Sorry, I haven't been doing a job (good or bad) maintaining this.

And thank you for doing a refresh!